rossum/password.py

```diff
@@ -23,3 +23,20 @@ def change_command(ctx: click.Context, password: str) -> None:
                 return click.echo(
                     'Please exit and run "rossum configure" to update existing credentials.'
                 )
+
+
+@cli.command(name="reset", help="Reset password. Please specify an email address.")
+@click.argument("email")
+@click.pass_context
+def reset_command(ctx: click.Context, email: str) -> None:
+    """
```

Comment:
Consider removing the description - it is not used in other commands. The commands should be self-descripting.